package.json

```diff
@@ -53,6 +46,8 @@
     "mocha": "^9.0.0",
     "ncp": "^2.0.0",
     "null-loader": "^4.0.0",
+    "protobufjs": "7.2.3",
+    "protobufjs-cli": "1.1.1",
```

Comment:
Do we use `pbjs` / `pbts` directly from GAX? I _think_ the `protobufjs-cli` dependency should be in `gapic-tools` only, but I might be wrong here.

Reply:
From what I can tell, we use it in the compile steps: [https://github.com/googleapis/gax-nodejs/blob/6d17ba2b459112cf6506262b765f3f3468a9cbcd/package.json#L75 (and the ones below)](https://github.com/googleapis/gax-nodejs/blob/6d17ba2b459112cf6506262b765f3f3468a9cbcd/package.json#L75-L81)

Reply:
Ah, this is `devDependency`, this is fine.